lcode2dPy/beam_generator/beam_generator.py

```diff
@@ -4,177 +4,135 @@
 from pandas import DataFrame  as df
 import numpy as np
 from numpy import sqrt, pi
+from ..config.config import find 
+from ..config.default_config import default_config 
+from .beam_profiles import distrs_from_shapes, get_segments_from_c_config, find_beam_profile_pars, split_into_segments
 
-# BEAM PROFILES
+particle_dtype2d = np.dtype([('xi', 'f8'), ('r', 'f8'),
```

Comment:
We work with x and y coordinates in 2d pusher, maybe we should remember them? 
At least it will be helpful for AQSA.